src/components/Panel.tsx

```diff
@@ -25,7 +25,7 @@ const getTimestamp = () => +new Date();
 
 export const DEFAULT_GROUP_ID = 'Other';
 
-const PanelWrapper = styled(({ children, className }) => (
+const PanelWrapper = styled(({ children, className }: any) => (
```

Comment:
Normally, I wouldn't add `any` like this, but since I'm just updating a dead repo for compatibility, I'm being a little looser.